### ISSUE TITLE
Asking registration model to translation which is not required to register

### ISSUE THREAD

**Issue by Tom1204** [deschler/django-modeltranslation#490]
Version: **django-modeltranslation==0.13b1**
models.py
```
class Speciality(DeleteModel, BaseModel):
    name = models.CharField(max_length=255)
    parent = models.ForeignKey('self', CASCADE, related_name='children', null=True)

    objects = SpecialityManager()

    def __str__(self):
        return self.name
------
class Region(DeleteModel, BaseModel):
    COUNTRY = 'country'
    REGION = 'region'
    CITY = 'city'

    TYPES = (
        (COUNTRY, 'Country'),
        (REGION, 'Region'),
        (CITY, 'City'),
    )
    parent = models.ForeignKey('self', PROTECT, related_name='sub_regions', null=True)
    name = models.CharField(max_length=255)
    type = models.CharField(max_length=20, choices=TYPES)

    cache = None

    def __str__(self):
        return self.name
```
translation.py
```
from modeltranslation.translator import TranslationOptions, translator

from main.models import Speciality, Region


class NameTranslationOption(TranslationOptions):
    fields = ('name',)


translator.register(Speciality, NameTranslationOption)
translator.register(Region, NameTranslationOption)
```
test.py
```
class BaseTest(TestAuthMixin, APITestCase):
    fixtures = [
        # TODO: write fixtures here
        'users.yaml',
        'languages.yaml',
        'files.yaml',
        'positions.yaml',
        'specialities.yaml',
        'staffs.yaml',
        'plans.yaml',
        'projects.yaml',
        'tasks.yaml',
        'skills.yaml',
        'search_work_statuses.yaml',
        'driver_licences.yaml',
        'regions.yaml',
        'institutions.yaml',
        'applicants.yaml',
        'employers.yaml',
        'vacancies.yaml',
    ]

class SpecialityTest(BaseTest):
    list_url = 'main:speciality-list'
    detail_url = 'main:speciality-detail'

    def test_create(self):
        data = dict(
            name_en='Stock manager',
            name_ru='Stock manager ru',
            name_uz='Stock manager uz',
            parent=2
        )

        response = self.client.post(reverse(self.list_url), data, format='json')

        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
        self.assertEqual(response.data['name_en'], data['name_en'])
        self.assertEqual(response.data['name_ru'], data['name_ru'])
        self.assertEqual(response.data['name_uz'], data['name_uz'])
        self.assertEqual(response.data['parent']['id'], data['parent'])
        return response.data['id']

    def test_update(self):
        pk = self.test_create()
        self.set_user()

        data = dict(
            name_en='Content manager',
            name_ru='Content manager ru',
            name_uz='Content manager uz',
            parent=1
        )

        response = self.client.put(reverse(self.detail_url, kwargs=dict(pk=pk)), data, format='json')
        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
        self.assertEqual(response.data['name_en'], data['name_en'])
        self.assertEqual(response.data['name_ru'], data['name_ru'])
        self.assertEqual(response.data['name_uz'], data['name_uz'])
        self.assertEqual(response.data['parent']['id'], data['parent'])
```
Asking translation option for other model Staff, giving error like below:
```Error
Traceback (most recent call last):
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/test/testcases.py", line 1012, in setUpClass
    call_command('loaddata', *cls.fixtures, **{'verbosity': 0, 'database': db_name})
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 148, in call_command
    return command.execute(*args, **defaults)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/modeltranslation/management/commands/loaddata.py", line 61, in handle
    return super(Command, self).handle(*fixture_labels, **options)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 72, in handle
    self.loaddata(fixture_labels)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 113, in loaddata
    self.load_label(fixture_label)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 177, in load_label
    obj.save(using=self.using)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/core/serializers/base.py", line 222, in save
    getattr(self.object, accessor_name).set(object_list)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 983, in set
    old_ids = set(self.using(db).values_list(self.target_field.target_field.attname, flat=True))
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/modeltranslation/manager.py", line 486, in values_list
    clone = self._values(*fields, prepare=True)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/modeltranslation/manager.py", line 451, in _values
    new_fields, translation_fields = append_fallback(self.model, original)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/modeltranslation/manager.py", line 89, in append_fallback
    opts = translator.get_options_for_model(model)
  File "/Users/Shared/SharedPycharmProjects/my_job/api/.venv/lib/python3.6/site-packages/modeltranslation/translator.py", line 652, in get_options_for_model
    'translation' % model.__name__)
modeltranslation.translator.NotRegistered: Problem installing fixture '/Users/Shared/SharedPycharmProjects/my_job/api/apps/core/fixtures/projects.yaml': The model "Staff" is not registered for translation
```
As I have printed condition results(```print(opts.registered, opts.related)```) in condition body, it shows False, False
```
    def get_options_for_model(self, model):
        """
        Thin wrapper around ``_get_options_for_model`` to preserve the
        semantic of throwing exception for models not directly registered.
        """
        opts = self._get_options_for_model(model)
        if not opts.registered and not opts.related:
            raise NotRegistered('The model "%s" is not registered for '
                                'translation' % model.__name__)
        return opts
```

Is there any solution?

**Comment by last-partizan** [comment]
Looks like #467 

I'm figuring out how to fix this.

**Comment by last-partizan** [comment]
It must be fixed in 0.13b3, please check

**Comment by Tom1204** [comment]
Thanks it is fixed
